metadata/en-US/changelogs/39.txt

```diff
@@ -0,0 +1 @@
+- Small bugfixes
```

Comment:
maybe be a bit more specific, like: "Fix crash on month and week view due to invalid rrule"?

Either way, please go ahead with the release :).